docs/man/rpmkeys.8.md

```diff
@@ -41,29 +41,25 @@ armored public key can be added to the **rpm** database using
 ring management is performed exactly like package management. For
 example, all currently imported public keys can be displayed by:
 
-**rpm -q gpg-pubkey**
-
-A more convenient way to display them is
-
 **rpmkeys** **\--list**
 
-More details about a specific public key, when imported, can be displayed by
-querying. Here\'s information about the Red Hat GPG/DSA key:
-
-**rpm -qi gpg-pubkey-db42a60e**
+This lists all currently imported public keys (aka certificates)
+by their fingerprint and User ID. The fingerprint is the handle
+used for all other operations on the keys.
 
-Finally, public keys can be erased after importing just like packages.
-Here\'s how to remove the Red Hat GPG/DSA key:
+**rpmkeys** **\--export** \[*FINGERPRINT \...*\]
 
-**rpmkeys** **\--export**
+Write the key(s) in an armor wrapped text format to standard out.
+This can be used to inspect the data with specialized tools, such
+as Sequoia or GnuPG, for example:
 
-Write the keys in an armor wrapped text format to standard out.
+**rpmkeys --export 771b18d3d7baa28734333c424344591e1964c5fc | sq inspect **
 
-**rpmkeys** **\--delete db42a60e**
+**rpmkeys** **\--delete** *FINGERPRINT \...*
 
-Or alternatively:
+Delete the keys in *FINGERPRINT*. For example:
```

Comment:
Suggestion: s/in/designated by/

Reply:
That's the word that was escaping me :smile: